client/src/pages/Landing/LandingFormSide.tsx

```diff
@@ -0,0 +1,33 @@
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './Styles/Landing';
+import LandingForm from './LandingForm';
+import Logo from '../../Images/logo.png';
+
+export default function Landing(): JSX.Element {
+  const classes = useStyles();
+
+  const handleSubmit = () => {
+    console.log('submited');
```

Comment:
Please remove console messages used for debugging

Reply:
done